tsdb/index/postings.go

```diff
@@ -57,13 +59,15 @@ var ensureOrderBatchPool = sync.Pool{
 type MemPostings struct {
 	mtx     sync.RWMutex
 	m       map[string]map[string][]storage.SeriesRef
+	lvs     map[string][]string
```

Comment:
Suggest to comment `m` and `lvs` here, also the rules on locking around `lvs`.

Reply:
Done in https://github.com/prometheus/prometheus/pull/15426/commits/e61c8c9e55fb499f172b668f72aeb80cf8e0785e